src/styles/quiz.scss

```diff
@@ -1,32 +1,117 @@
-// @import "variables", "styles";
-
-#questionImage,
-#progressImage {
-  border-color: var(--border-color);
-  border-radius: 10px;
-  border-style: outset;
+#quiz {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  max-width: 800px;
+  margin: 0 auto;
 }
 
-#progressImage {
-  border-style: inset;
+#questionText {
+  width: 100%;
+  padding: 20px;
+  margin-bottom: 20px;
+  background-color: rgba(0, 0, 0, 0.6); 
+  color: white;
+  font-size: 1.5rem;
+  text-align: center;
+  border-radius: 8px; 
 }
 
 .choices {
-    align-items: center;
-    display: flex;
-    flex-direction: column;
-    gap: 10px;
-    justify-content: center;
-
-      button {
-        max-width: 300px;
-      }
+  display: grid;
+  gap: 10px;
+  width: 100%;
+  grid-template-columns: repeat(2, 1fr);
+  margin-bottom: 20px; 
+
+  @media (max-width: 600px) {
+    grid-template-columns: 1fr;
   }
 
-  @media screen and (min-width: 768px) {  
-    .choices {
-      display: grid;
-      grid-template-columns: repeat(2, auto);
-      gap: 10px;
+  @media (min-width: 601px) and (max-width: 900px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+.MuiButtonBase-root {
+  position: relative;
+  height: 200px;
+  width: 100%;
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(0, 0, 0, 0.1);
+  transition: background-color 0.3s;
+
+  &:hover, &.Mui-focusVisible {
+    z-index: 1;
+    .MuiImageBackdrop-root {
+      opacity: 0.15;
     }
-  }
\ No newline at end of file
+    .MuiImageMarked-root {
+      opacity: 0;
+    }
+    .MuiTypography-root {
+      border: 4px solid currentColor;
+    }
+  }
+}
+
+.image-bg {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-size: cover;
+  background-position: center;
+  z-index: 0;
+}
+
+.image-overlay {
+  position: relative;
+  z-index: 1;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: rgba(0, 0, 0, 0.5); 
+  color: white;
```

Comment:
Different schools of thought so do what you want but I'd recommend trying to stay consistent and use variables/hex codes across the board rather than slipping named colors in. Super nit picky, though

Reply:
That line was a mistake. I had pressed to find out the specific color to use in something else. I will change it back.